gems/pending/util/mount/miq_generic_mount_session.rb

```diff
@@ -63,12 +63,7 @@ def self.uri_scheme_to_class(uri)
   end
 
   def mount_share
-    # Check if settings contains the mount_point to use
-    @mnt_point = @settings[:mount_point].blank? ? File.join(self.class.base_mount_point, "miq_#{MiqUUID.new_guid}") : @settings[:mount_point]
-    raise MiqException::MountPointAlreadyExists, "#{@mnt_point} directory already exists!!!" if File.exist?(@mnt_point)
-
-    FileUtils.mkdir_p @mnt_point
-    raise MiqException::MiqLogFileMountPointMissing, "mount point: [#{@mnt_point}] failed to be created" unless File.directory?(@mnt_point)
+    @mnt_point = settings_mount_point || Dir.mktmpdir
```

Comment:
Cool, I see `Dir.mktmpdir` works but when was that added?  I don't see it in ruby-doc: http://ruby-doc.org/core-2.2.2/Dir.html